tests/page-templates/18-usertiming/05-usertiming-second-beacon-no-performance-now.html

```diff
@@ -0,0 +1,32 @@
+<%= header %>
+<%= boomerangSnippet %>
+<script>
+	window.performance.now = undefined;
+</script>
+<script src="05-usertiming-second-beacon-no-performance-now.js" type="text/javascript"></script>
+<script src="../../vendor/usertiming-compression/src/usertiming-decompression.js" type="text/javascript"></script>
+<script>
+	if (BOOMR_test.isUserTimingSupported()) {
+		window.performance.mark("pre-load-mark-start");
+		window.performance.mark("pre-load-mark-end");
+		window.performance.measure("pre-load-measure", "pre-load-mark-start", "pre-load-mark-end");
+	}
+
+	function sendPostLoadMark() {
+		if (BOOMR_test.isUserTimingSupported()) {
+			window.performance.mark("post-load-mark-start");
+			window.performance.mark("post-load-mark-end");
+			window.performance.measure("post-load-measure", "post-load-mark-start", "post-load-mark-end");
+			BOOMR.responseEnd("test");
+		}
+	}
+
+	BOOMR_test.init({
+		testAfterOnBeacon: 2,
```

Comment:
Now that i think of it, we should do `BOOMR_test.isUserTimingSupported() ? 2 : 1` here. The 2nd beacon won't get sent by `sendPostLoadMark` if UT is not supported. Same for other test page.